.github/workflows/build.yml

```diff
@@ -62,7 +62,6 @@ jobs:
 
   publish:
     needs: build
-    if: ${{ github.event_name == 'workflow_dispatch' || (github.event_name == 'push' && startsWith(github.ref_name, 'release')) }}
```

Comment:
This was done on purpose

Reply:
what is the purpose?

Reply:
it was decided to add this restriction because without it we had too many Images and it made it hard to find the right one.
Given that we do not use these images (only release and sometimes main), it is logical to remove the extra action, it trigger it manually for rare cases when it needed

Reply:
but this rule still exists
```
on:
  push:
    branches:
      - 'main'
      - 'release-*'
  pull_request:
    types:
      - opened
      - reopened
      - ready_for_review
      - synchronize
 ```
so it will not be triggered for dev branches